module/Frontpage/view/frontpage/frontpage/home.phtml

```diff
@@ -0,0 +1,23 @@
+<div class="jumbotron">
+    <h2><?= $this->translate('Study association GEWIS') ?></h2>
+    <p><?= sprintf($this->translate('Welcome to the website of Study association %sGEWIS%s. This is just a bootstrap thingy for now, until the design is implemented.'), '<span class="gewis-red">', '</span>') ?></p>
+    <p><a class="btn btn-success" href="https://github.com/GEWIS/gewisweb" target="_blank"><?php echo $this->translate('Fork the GEWIS website on GitHub') ?> &raquo;</a></p>
+</div>
```

Comment:
In the new design, this Jumbotron is not on this page anymore, it is included in the layout itself.